src/herder/HerderTests.cpp

```diff
@@ -679,8 +679,7 @@ TEST_CASE("whitelist", "[herder]")
 
         DataValue value;
         value.resize(4);
-        SignatureHint hint =
-            SignatureUtils::getHint(accountWL.getPublicKey().ed25519());
+        SignatureHint hint = SignatureUtils::getHint(accountWL.getPublicKey().ed25519());
```

Comment:
is this intentional? lets not create a diff with the original code unless we really want to.

Reply:
Yes.  I am trying to maintain the overall formatting rules for the code base, which includes a hard limit of 80 characters per line.  Along with this, I am fighting Xcode's desire to indent differently.  In this instance, moving the rhs of the assignments satisfied both.